types/rsocket-core/RSocketMachine.d.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  CancelFrame,
+  ConnectionStatus,
+  DuplexConnection,
+  Frame,
+  FrameWithData,
+  Payload,
+  Responder,
+  PartialResponder,
+  ReactiveSocket,
+  RequestFnfFrame,
+  RequestNFrame,
+  RequestResponseFrame,
+  RequestStreamFrame,
+  RequestChannelFrame,
+  ISubject,
+  ISubscription,
+  ISubscriber
+} from 'rsocket-types';
+
+import { Flowable, FlowableProcessor, Single } from 'rsocket-flowable';
+import {
+  createErrorFromFrame,
+  getFrameTypeName,
+  isComplete,
+  isNext,
+  isRespond,
+  CONNECTION_STREAM_ID,
+  ERROR_CODES,
+  FLAGS,
+  FRAME_TYPES,
+  MAX_REQUEST_N,
+  MAX_STREAM_ID,
+} from './RSocketFrame';
+import { PayloadSerializers, IdentitySerializers } from './RSocketSerialization';
+
+export type Role = 'CLIENT' | 'SERVER';
+
+export class ResponderWrapper<D, M> implements Responder<D, M> {
+  _responder: PartialResponder<D, M>;
```

Comment:
A `_` prefix often indicates a "private" or "protected" member. Is this intended to be public?

Reply:
No, I wasn't aware that the typings should only be for the public members. Will review all the definitions and revert.